hieradata/env/production.yaml

```diff
@@ -30,6 +38,7 @@ profile::kubernetes::params::clusters:
+      ldap_public_ip: 52.232.180.203
```

Comment:
This looks like hard-coded, can you add a comment where this hard-coded value comes from? Presumably terraform?